js/EventTarget.js

```diff
@@ -1,23 +1,17 @@
 const { isFunction } = require('./lib/utils.js');
+const { kNapiObj } = require('./lib/symbols.js');
 
 const kEventListeners = Symbol('node-web-audio-api:event-listeners');
 const kDispatchEvent = Symbol.for('node-web-audio-api:napi-dispatch-event');
 
-module.exports = (superclass, eventTypes = []) => class EventTarget extends superclass {
+class EventTarget {
```

Comment:
Would it be possible to use the native EventTarget with the new wrapping? This would make it even more spec compliant.

Reply:
That would make sens indeed, but it requires some work / testing to bind that properly with the native Rust events. I will open a separate issue

Reply:
Done in #[3bf361e](https://github.com/ircam-ismm/node-web-audio-api/pull/99/commits/3bf361e861843f97f77debb27b41ecd6885c25d7)